x-pack/plugins/infra/public/components/asset_details/tabs/overview/metrics/metrics_grid.tsx

```diff
@@ -4,77 +4,57 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
```

Comment:
Q: I am wondering how this change is affecting the Kpis? I am searching for the usage somewhere but I couldn't find any 🤔

Reply:
It's not by affected by the Kpi, but by the changes in the `use_lens_attributes`. The main difference is the creation of `Chart` component, which encapsulates the usage of `LensChart` and memoizes some of its props, fixing some re-rendering problems.

Another thing is splitting what was `section` related component and `grid` related component

Reply:
The `ChartGrid` had a map that directly called the `LensChart` component. That is the main change, I would say.

Reply:
Ok, thanks for explaining wanted to make sure that I checked everything changed here 🙂